Makefile

```diff
@@ -314,6 +322,23 @@ OPM = $(shell which opm)
 endif
 endif
 
+.PHONY: yq
+YQ = ./bin/yq
+yq: ## Download yq locally if necessary.
+ifeq (,$(wildcard $(YQ)))
+ifeq (,$(shell which yq 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(dir $(YQ)) ;\
+	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
```

Comment:
why is it needed? you are downloading binary to be run, so can be put anywhere on PATH, right?

Reply:
yq is for manipulating the CSV YAML file. 
it is downloaded to `./bin` as other tools do.